src/test/java/MarkTestStatus.java

```diff
@@ -0,0 +1,71 @@
+import okhttp3.*;
+import org.json.JSONObject;
+import org.testng.annotations.Test;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Base64;
+import java.util.Properties;
+
+public class MarkTestStatus {
+    @Test
+    public void test() throws Exception {
+        String status = getPercyBuildStatus();
+        if(status.equals("finished")){
+            setSessionStatus("passed","Percy Build has been successfully completed!");
+        }else{
+            setSessionStatus("failed","Percy Build has failed!");
+        }
+    }
+    public String getPercyBuildStatus() throws IOException {
+        String buildID = System.getProperty("buildid");
+        OkHttpClient client = new OkHttpClient();
+
+        Request request = new Request.Builder()
+                .url("https://percy.io/api/v1/builds/"+buildID)
+                .get()
+                .addHeader("Authorization", "Token "+System.getenv("PERCY_TOKEN"))
+                .build();
+
+        Response response = client.newCall(request).execute();
+
+        JSONObject jsonObject = new JSONObject(response.body().string());
+        JSONObject jsonpObject2 = new JSONObject(jsonObject.get("data").toString());
```

Comment:
If Build is not found this like will throw following exception
org.json.JSONException: JSONObject["data"] not found.
        at org.json.JSONObject.get(JSONObject.java:570)
        at MarkTestStatus.getPercyBuildStatus(MarkTestStatus.java:34)
        at MarkTestStatus.test(MarkTestStatus.java:14)
lets add null checks?

Reply:
added checks